docs/console/company-configuration/providers/extensions/orchestrator-generator.md

```diff
@@ -0,0 +1,270 @@
+---
+id: orchestrator-generator
+title: Orchestrator Generator
+sidebar_label: Orchestrator Generator
+---
+
+![Orchestrator Generator](img/orchestrator-generator.png)
+
+:::info
+
+An Orchestrator Generator can only be used when the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled.
+
+:::
+
+One of the main features of Mia-Platform is the automatic generation of the **orchestrator's files**, ready to **be deployed**; these files are saved in the **configuration repository**. For instance, the Console could generate Kubernetes files.
+
+It could be that you would want to generate those files differently: if this is the case, you should use an **Orchestrator Generator**.
+When the Orchestrator Generator Extension is enabled, the Console sends a **POST request** to its Provider's API Base URL to generate the deploy files. The request body contains some information about the Project and the Configuration, and the Console expects to receive, in response, the files.
```

Comment:
```suggestion
When a Provider with the Orchestrator Generator capability is enabled, the Console sends a **POST request** to the Provider API Base URL to generate the deploy files. The request body contains some information about the Project and the Configuration, and the Console expects to receive, in response, the files to be saved and deployed.
```

Reply:
is the Provider enabled, or is the Extension enabled by selecting a Provider?

Reply:
interesting point; let's keep the way you wrote